packages/freezed_annotation/lib/freezed_annotation.dart

```diff
@@ -151,6 +151,48 @@ class FreezedWhenOptions {
   final bool? maybeWhen;
 }
 
+class _FreezedWhenOptionsConverter
+    implements JsonConverter<FreezedWhenOptions?, Object?> {
+  const _FreezedWhenOptionsConverter();
+
+  @override
+  FreezedWhenOptions? fromJson(Object? json) {
+    if (json == true || json == 'all') return FreezedWhenOptions.all;
+    if (json == false || json == 'none') return FreezedWhenOptions.none;
```

Comment:
If `true/false` is identical to `all/none`, why have string variants when bool variants are enough?

Reply:
I felt like all/none was easier to understand as it's conceptually closer to what being done in dart, though I can remove the string variants if needed.

Reply:
I'd remove them for now. I don't think that's needed for now; let's keep it simple